src/codeu/chat/client/ClientUser.java

```diff
@@ -84,6 +82,7 @@ public boolean signInUser(String name, String password) {
 	    return (prev != current);
     }
 
+
     System.out.println("Login was UNSUCCESSFUL due to your username and password combination of: " + name + " " + password);
```

Comment:
We should have this go into LOG. And we probably shouldn't be displaying a password lol